src/main/kotlin/com/sourcegraph/cody/edit/FixupService.kt

```diff
@@ -70,23 +69,27 @@ class FixupService(val project: Project) : Disposable {
       logger.warn("Edit code invoked when Cody not enabled")
       return false
     }
-    if (CodyStatusService.getCurrentStatus(project) == CodyStatus.CodyAgentNotRunning) {
+
+    if (!CodyAgentService.isConnected(project)) {
```

Comment:
This is a bit more explicit/correct, as it is vased on real state and not the status reported on the UI.